vnext/src/Libraries/Components/View/ReactNativeViewAttributes.windows.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+'use strict';
+import ReactNativeStyleAttributes from './ReactNativeStyleAttributes';
+
+const UIView = {
```

Comment:
What are your plans for TS typings?

Reply:
I don't have any, open to suggestions that we could hopefully handle in a future PR / contribution elsewhere.

Reply:
I updated #9053, adding a note that the TS typings will also need to be updated.

Reply:
@types/react-native is pretty open to contributions. I've added some desktop only bits there before.

I guess we don't have typings for `onMouseEnter`, etc on view there yet... Seeing the any casts for usage there make me worry a bit, for anyone who will add the API if we think we will ever replace it with something different upstream.

Reply:
Oh wait no, upstream does have those props, that were added for web IIRC. https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/Libraries/Components/View/ViewPropTypes.js#L86

```js
type MouseEventProps = $ReadOnly<{|
  onMouseEnter?: (event: MouseEvent) => void,
  onMouseLeave?: (event: MouseEvent) => void,
|}>;
```